src/components/map/map.tsx

```diff
@@ -1,45 +1,36 @@
-import React, { useRef, useEffect, useState, useCallback } from 'react';
+import React, { useRef, useEffect } from 'react';
 import mapboxgl from 'mapbox-gl';
 import style from './map.module.css';
 import 'mapbox-gl/dist/mapbox-gl.css';
 import { mapboxData } from '@atb/modules/org-data';
-import { isFeaturePoint, isStopPlace } from './utils';
 import Button from '../button/button';
 import { MonoIcon } from '@atb/components/icon';
 import { ComponentText, useTranslation } from '@atb/translations';
 import { FocusScope } from '@react-aria/focus';
+import { MapLegType } from '@atb/page-modules/departures/server/journey-planner/validators';
```

Comment:
Should probably not import this?

Reply:
I'd we are to import from page-modules (which we should try to avoid), we should import from the server path based on what Mikael have said.

Reply:
`@atb/page-modules/departures/server` that is?

Reply:
Yes. But are we able to use an unnamed type with only the properties we use or have a fragment type?